packages/be-gateway/src/routes/setting/index.ts

```diff
@@ -0,0 +1,30 @@
+import { Router } from 'express'
```

Comment:
đổi toàn bộ sang kiểu viết class, chỉ những file cũ, chưa đổi được mới phải viết dạng router.get như này thôi